cli/functionary/package.py

```diff
@@ -82,20 +89,22 @@ def publish(ctx, path, host):
 
     full_path = pathlib.Path(path).resolve()
     tarfile_name = full_path.joinpath(f"{full_path.name}.tar.gz")
-
-    with tarfile.open(str(tarfile_name), "w:gz") as tar:
-        tar.add(str(full_path), arcname=".")
+    with tarfile.open(str(tarfile_name), "w:gz") as tar: 
+        tar.add(str(full_path), arcname='')
 
     click.echo(f"Publishing {str(tarfile_name)} package to {host}")
 
     # publish should http the tar to a server, wait for return
     upload_file = open(tarfile_name, "rb")
     upload_response = None
-    headers = {"Authorization": f"Bearer {token}"}
-
+    environment_id = str(getEnvironmentID())
+    print('Token ' + token)
+    print('ID ' + environment_id)
+    headers = {"Authorization": f"Token {token}", "X-Environment-ID": f"{environment_id}"}
+    publish_host = host + "/api/v1/publish"
```

Comment:
I'm really picking nits here, but this should probably be `publish_url` rather than `publish_host` since it consists of more than just the host info.